FrameworkSystem/DB/ProxyDB.py

```diff
@@ -1066,7 +1066,7 @@ def setPersistencyFlag(self, userDN, userGroup, persistent=True):
         return result
       userName = result['Value']
       cmd = "INSERT INTO `ProxyDB_Proxies` ( UserName, UserDN, UserGroup, Pem, ExpirationTime, PersistentFlag ) VALUES "
-      cmd += "( %s, %s, %s, '', UTC_TIMESTAMP(), 'True' )" % (userName, sUserDN, sUserGroup)
+      cmd += "( '%s', %s, %s, '', UTC_TIMESTAMP(), 'True' )" % (userName, sUserDN, sUserGroup)
```

Comment:
why would there be quotes around username, but not the dn or the group ?

Reply:
Because sUserDN, sUserGroup already contain quotes.

Reply:
Why not use `self._escapeString` like is done for `sUserDN` and `sUserGroup`?

Reply:
changed to use _escapeString